create_item.rb

```diff
@@ -78,7 +84,6 @@ def create_book
     author = gets.chomp
 
     new_book(title, author)
-
     puts('Book created successfully')
     run
   end
```

Comment:
- [OPTIONAL] I noticed that the new_book doesn't store in the books.json file, I think you forgot to use `save_book(@book)`, so kindly fix that. I'll mark it as optional but I strongly recommend fixing this issue 💯👌
```
  def new_book(title, author)
    new_book = Book.new(title, author)
    @book << new_book
    save_book(@book)
  end
```
Your Code:  
```
  def new_book(title, author)
    new_book = Book.new(title, author)
    @book << new_book
  end
```